sdk/cosmosdb/cosmos/test/public/functional/excludedLocationsRequest.spec.ts

```diff
@@ -0,0 +1,996 @@
+      includedPaths: [{ path: "/*" }],
+      excludedPaths: [{ path: '/"_etag"/?' }],
+    },
+    partitionKey: {
+      paths: ["/_partitionKey"],
+      kind: "Hash",
+    },
+    conflictResolutionPolicy: {
+      mode: "LastWriterWins",
+      conflictResolutionPath: "/_ts",
+      conflictResolutionProcedure: "",
+    },
+    geospatialConfig: { type: "Geography" },
+    _rid: "kdY4AIn8g54=",
+    _ts: 1572274839,
+    _self: "dbs/kdY4AA==/colls/kdY4AIn8g54=/",
+    _etag: '"00007500-0000-0100-0000-5db702970000"',
+    _docs: "docs/",
+    _sprocs: "sprocs/",
+    _triggers: "triggers/",
+    _udfs: "udfs/",
+    _conflicts: "conflicts/",
+  },
+  code: 200,
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const ServiceUnavailableResponse = {
+  code: StatusCodes.ServiceUnavailable,
+  result: {},
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const TimeoutResponse = {
+  code: TimeoutErrorCode,
+  result: {},
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const SuccessCreateResponse = {
+  code: StatusCodes.Created,
+  result: {},
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const SuccessReadResponse = {
+  code: StatusCodes.Ok,
+  result: { Documents: [] as any[] },
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const partitionKeyRangesResponse = {
+  code: StatusCodes.Ok,
+  result: {
+    _rid: "kdY4AIn8g54=",
+    PartitionKeyRanges: [
+      {
+        id: "0",
+        minInclusive: "",
+        maxExclusive: "FF",
+        ridPrefix: 0,
+        _self: "dbs/kdY4AA==/colls/kdY4AIn8g54=/pkranges/kdY4AIn8g54CAAAAAAAAAA==/",
+        throughputFraction: 1,
+        status: "online",
+        parents: [],
+        _ts: 1572274839,
+      },
+    ],
+    _count: 1,
+  },
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const changeFeedResponse = {
+  code: StatusCodes.Ok,
+  result: {
+    Documents: [
+      { id: "doc1", _partitionKey: "pk1" },
+      { id: "doc2", _partitionKey: "pk2" },
+    ],
+    _count: 2,
+  },
+  headers: {
+    "x-ms-continuation": "continuation-token",
+    etag: '"etag-value"',
+  },
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const changeFeedEmptyResponse = {
+  code: StatusCodes.NotModified,
+  result: {
+    Documents: [],
+    _count: 0,
+  },
+  headers: {},
+  diagnostics: getEmptyCosmosDiagnostics(),
+};
+
+const options: CosmosClientOptions = {
+  endpoint,
+  key: masterKey,
+  connectionPolicy: {
+    preferredLocations: ["East US", "Australia East", "West US"],
+  },
+};
+
+const options1: CosmosClientOptions = {
+  endpoint,
+  key: masterKey,
+};
+
+const options2: CosmosClientOptions = {
+  endpoint,
+  key: masterKey,
+  connectionPolicy: {
+    preferredLocations: ["East US", "Australia East"],
+  },
+};
+
+describe("Excluded Region tests", { timeout: 30000 }, () => {
+  it("Request-level excludedLocations should override client-level for READ", async () => {
```

Comment:
Test case names are misleading. We do not have client level excludedLocations anymore